sync3/request.go

```diff
@@ -260,6 +260,11 @@ func (rl *RequestList) WriteSwapOp(
 	}
 }
 
+// HasBumpEventTypes returns true iff this request has specified bump_event_types.
+func (rl *RequestList) HasBumpEventTypes() bool {
+	return rl.BumpEventTypes != nil && len(rl.BumpEventTypes) > 0
```

Comment:
The nil check is superfluous here, as a nil slice has length 0.

Reply:
Oh true enough. /me inlines